panoramapublic/resources/queries/panoramapublic/proteinSearch.sql

```diff
@@ -0,0 +1,16 @@
+PARAMETERS
+(
+    proteinLabel VARCHAR,
+    exactMatch BIT default false
+)
+
+SELECT created,
+       count(proteinLabel) as matches,
+       title,
+       organism,
+       citation,
+       pxid,
+       proteinLabel AS proteinLabel,
```

Comment:
Same question here - don't you need to know the value for exactMatch to do the same thing on the next page?

Reply:
added